omero/sysadmins/unix/install-web.txt

```diff
@@ -303,6 +239,32 @@ OMERO.web Maintenance
 Customizing your OMERO.web installation
 ---------------------------------------
 
+By default OMERO.web expects to be run from the root URL of the web server.
+This can be changed by setting :property:`omero.web.force_script_name` and
+:property:`omero.web.static_url`. For example, to make OMERO.web appear at
+`http://example.org/omero/`:
+
+   ::
+
+      $ bin/omero config set omero.web.force_script_name '/omero'
```

Comment:
The indenting here and below is causing that annoying purple highlighting in plone - can you move the `::` back against the margin and just indent the content below 3 spaces please?